1.8/administration/installing/cloud/aws/advanced/aws-ami.md

```diff
@@ -0,0 +1,25 @@
+---
+post_title: Installing Using a Custom AMI
+nav_title: Custom AMI
+menu_order: 199
+---
+
+You can use customized [Amazon Machine Images (AMI)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) based on CentOS 7 or CoreOS to launch DC/OS with the advanced templates. 
+
+- A custom AMI can be used to integrate DC/OS installation with your own in-house configuration management tools.
+- A custom AMI can be used if you want kernel or driver customization.
+
+To get started, build a custom AMI and then install DC/OS by using the advanced templates. 
+
+# Build a custom AMI
+Here is the recommend method to build your AMI:
+
+1.  Using the DC/OS [cloud_images](https://github.com/dcos/dcos/tree/master/cloud_images) scripts for CentOS 7 as a template, create an AWS AMI in the same target region as your DC/OS stack. 
+
+    **Important:** The AMI must satisfy all of the DC/OS AMI prerequisites as shown in the template.
+
+1.  Build and deploy your AMI. For an example of how to build and deploy AMIs to multiple regions, see the DC/OS Packer build scripts [here](https://github.com/dcos/dcos/blob/master/cloud_images/centos7/packer.json).
+
+# Install DC/OS
+
+Launch the DC/OS advanced template on CloudFormation and specify your custom AMI. For more information, see the QuickStart [documentation](/docs/1.8/administration/installing/cloud/aws/advanced/quickstart/#launch).
```

Comment:
We are overloading the term `template`... I made a bunch of changes to try and address that and a few other issues. Here is a suggested update to the section above (starting with Build a custom AMI).

```
1. Start with the DC/OS [cloud_images](https://github.com/dcos/dcos/tree/master/cloud_images) scripts which build a CentOS7 AMI with all of the DC/OS pre-requisites installed. First make sure that you can build and deploy an AMI using these scripts as is. An AMI must be deployed to every region where a cluster will be launched. The DC/OS Packer build script [create_dcos_ami.sh](https://github.com/dcos/dcos/blob/master/cloud_images/centos7/create_dcos_ami.sh) can deploy the AMI to multiple regions by setting the environment variable `DEPLOY_REGIONS` before running the script.

1. Launch the DC/OS advanced template using the AWS CloudFormation web console and specify your custom AMI. Be sure to verify that the cluster launched successfully. For more information, see the QuickStart [documentation](/docs/1.8/administration/installing/cloud/aws/advanced/quickstart/#launch).

1. Once you are comfortable building and deploying an AMI, adjust the scripts to add your own AMI customizations. Note that all of the DC/OS prerequisites must still be met in order for the AMI to work with DC/OS.

1. Repeat the testing process by launching a cluster with the newly built AMI. Verify that DC/OS starts as expected and that services can be launched on the cluster. To troubleshoot failures it will be necessary to have a good understanding of the DC/OS service startup process and have SSH access to all of the cluster nodes. The DC/OS Slack community may be a good place to get help.
```